cmd/nerdctl/pull.go

```diff
@@ -103,10 +114,92 @@ func pullAction(cmd *cobra.Command, args []string) error {
 		}
 		_, err = ipfs.EnsureImage(ctx, client, ipfsClient, cmd.OutOrStdout(), cmd.ErrOrStderr(), snapshotter, scheme, ref,
 			"always", ocispecPlatforms, unpack, quiet)
-		return err
+		if err != nil {
```

Comment:
This change seems causing CI failure.

Probably we should raise an error when ipfs && cosign

Reply:
I don't understand what should I do? Can you explain a bit more?

Reply:
We can add cosign support on ipfs through following-up PRs. 
As of now, we should return an error when `verify != "none"` and the ref is an IPFS ref (i.e. we get `err == nil` [here](https://github.com/containerd/nerdctl/blob/a057cf7d0b5d9e32ca66f2a168fa19b542fa24bb/cmd/nerdctl/pull.go#L110)).

Reply:
I've updated the according to your feedback 9e2ac78. Is that ok?

Reply:
Can we have the check just after `if scheme, ref, err := referenceutil.ParseIPFSRefWithScheme(args[0]); err == nil` ?

Reply:
of course, 5af0f22.

Reply:
sorry about the mess, I couldn't understand at the beginning :/